pipeline-templates/prepare-signing.yaml

```diff
@@ -14,6 +14,9 @@ steps:
       signType: ${{ parameters.SignType }}
       zipSources: false
       feedSource: https://dnceng.pkgs.visualstudio.com/_packaging/MicroBuildToolset/nuget/v3/index.json
+      azureSubscription: 'MicroBuild Signing Task (DevDiv)'
+      ${{ if eq(variables['Build.Reason'], 'IndividualCI') }}:
+        ConnectedPMEServiceName: c24de2a5-cc7a-493d-95e4-8e5ff5cad2bc

```

Comment:
Is this the only change related to PME stuff?

Reply:
In a way, yes. On the same day PME is enforced, they are also enforcing no manual builds, so this is making the manual stuff not manual. It's required to always real sign on the PME pipeline so that is required for this pr to work as well